### PR TITLE
Inst UI 3563 color preset color mixer axe check tests failing

### PR DESCRIPTION
**Pull request by NamNguyenInstructure**

### CODE REVIEW

File: packages/ui-color-picker/src/ColorPreset/__tests__/ColorPreset.test.tsx [hunk 1]
```diff
@@ -359,14 +353,5 @@ describe('<ColorPreset />', () => {
...
-  // describe('should be accessible', () => {
-  //   generateA11yTests(ColorPreset, ColorPresetExamples)
-  //   it('a11y', async () => {
-  //     await mount(<ColorPreset {...testValue} />)
-  //     const subject = await ColorPresetLocator.find()
-  //
-  //     expect(await subject.accessible()).to.be.true()
-  //   })
-  // })
+  // The accessibility tests are ignored because the tooltips of the ColorIndicator, which are defined in the "aria-labelledby" attribute, are located out of the scope of the ColorPreset.
```
Comment (ke1k0) [inline packages/ui-color-picker/src/ColorPreset/__tests__/ColorPreset.test.tsx:356]: I checked this component with the a11y checker tool, and it indeed throws tons of errors... Well, the good news is that all these errors are coming from the fact that we are nesting the drilldown and tooltip components, the same element is the trigger for both, so we should look for a workaround here...

@HerrTopi what did the a11y audit say? I'm pretty sure they found these errors too, but were they willing to ignore them?

image

File: packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx [hunk 2]
```diff
@@ -235,6 +235,7 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
         onKeyDown={(e) => this.handleKeyDown(e)}
         onMouseDown={(e) => this.handlePaletteMouseDown(e)}
         aria-label={this.props.navigationExplanationScreenReaderLabel}
+        role="dialog"
```
[the remark this answers is not in the document]
Comment (HerrTopi) [reply packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx:238]: Yeah, this workaround seems fine.

File: packages/ui-color-picker/src/ColorPreset/__tests__/ColorPreset.test.tsx (hunk 1 above)
Comment (HerrTopi) [reply packages/ui-color-picker/src/ColorPreset/__tests__/ColorPreset.test.tsx:356]: I honestly don't remember, but I think we should ask for a new audit anyways

**Review by ke1k0 — CHANGES_REQUESTED** [review]

File: packages/ui-color-picker/src/ColorMixer/Slider/index.tsx [hunk 3]
```diff
@@ -217,10 +217,10 @@ class Slider extends Component<SliderProps> {
         tabIndex={this.props.disabled ? undefined : 0}
         aria-label={this.props.navigationExplanationScreenReaderLabel}
         // TODO: check RangeInput and ProgressBar to see how the slider role can be handled well
-        // role="slider"
-        // aria-valuemin={this.props.minValue}
-        // aria-valuemax={this.props.maxValue}
-        // aria-valuenow={this.roundedValue}
+        role="slider"
```
[the remark this answers is not in the document]
Comment (NamNguyenInstructure) [reply packages/ui-color-picker/src/ColorMixer/Slider/index.tsx:219]: Yes I will

[the remark this answers is not in the document]
Comment (NamNguyenInstructure) [reply packages/ui-color-picker/src/ColorMixer/Slider/index.tsx:219]: Jessica was happy with these changes 👍

File: packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx [hunk 4]
```diff
@@ -234,8 +238,11 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
         tabIndex={this.props.disabled ? undefined : 0}
         onKeyDown={(e) => this.handleKeyDown(e)}
         onMouseDown={(e) => this.handlePaletteMouseDown(e)}
-        aria-label={this.props.navigationExplanationScreenReaderLabel}
+        id={this._id}
       >
+        <ScreenReaderContent>
```
Comment (ke1k0) [inline packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx:243]: As we discussed on the standup: please ask Jes about this :)

Comment (NamNguyenInstructure) [reply packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx:243]: I've just asked her

**Comment by ke1k0** [comment]
Sry, I might have missed it, but when did she say it is not needed? My last info is that we have to revert one part of it, but the rest of the changeset is working good and is needed.

**Comment by NamNguyenInstructure** [comment]
I misunderstand when you said it need to be reverted, I restore it now

**Pull request closed without merging**